examples/platform_view/lib/main.dart

```diff
@@ -47,6 +47,17 @@ class _MyHomePageState extends State<MyHomePage> {
     });
   }
 
+  Text _getButtonText() {
```

Comment:
nit: since this function has no parameter list, you could make it a static getter.

```dart
  static Widget get buttonText {
    // ... code here
  }

```